tdms/tests/unit/array_tests/test_DTilde.cpp

```diff
@@ -53,6 +53,7 @@ void DTildeTest::test_incorrect_number_of_fields() {
     REQUIRE_THROWS_AS(dt.initialise(matlab_input, 0, 0), runtime_error);
   }
   SECTION("Struct with too few fields") {
+    SKIP("Causes segmentation violation.");
```

Comment:
#191 is still at large. Although I think removing MATLAB will fix this as the weird MATLAB pointers will be cleaned up.

Reply:
Needed to use `SKIP` a lot earlier than I intended 🙃